cpp/open3d/core/EigenConverter.h

```diff
@@ -50,6 +50,26 @@ core::Tensor EigenMatrixToTensor(const Eigen::Matrix<T, M, N, A> &matrix) {
                         dtype);
 }
 
+template <class T>
```

Comment:
@theNded I have already added TensorToEigenMatrix ops in master, can you pull that and add your changes, if you want any?